static/styles/zulip.css

```diff
@@ -1090,6 +1090,16 @@ td.pointer {
     cursor: pointer;
 }
 
+.password-div {
+    position: relative;
+
+    .password_visibility_toggle {
+        position: absolute;
+        right: 10px;
+        top: 40px;
+    }
+}
+
```

Comment:
Should this be in `settings.css`?  It also seems like a duplicate of the `settings.css` copy above.